docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1268,29 +1250,11 @@ subject, [`.and('have.attr')`](/api/commands/and) is executed against the same
 element. This is handy when you need to assert multiple things against a single
 subject quickly.
 
-If we wrote this assertion in the explicit form "the long way", it would look
-like this:
-
-```js
-cy.get('tbody tr:first').should(($tr) => {
-  expect($tr).to.have.class('active')
-  expect($tr).to.have.attr('href', '/users')
-})
-```
-
-The implicit form is much shorter! So when would you want to use the explicit
-form?
-
-Typically when you want to:
-
-- Assert multiple things about the same subject
-- Massage the subject in some way prior to making the assertion
-
-### Explicit Subjects
+### Mocha Assertions
 
-Using `expect` allows you to pass in a specific subject and make an assertion
-about it. This is probably how you're used to seeing assertions written in unit
-tests:
+Using `expect` allows you to assert on any javascript object, not just the
```

Comment:
```suggestion
Using `expect` allows you to assert on any JavaScript object, not just the
```

Reply:
Fixed in [`939111a` (#5315)](https://github.com/cypress-io/cypress-documentation/pull/5315/commits/939111ab65ce85768e102f6b7c40bb2d4f355b5e)